### DISCUSSION TITLE
DLS 2.0 is different from the previous DLS

### DISCUSSION THREAD

**Discussion by dotdesign** [TDRA-ae/aegov-dls#56] [General]
I'm just curious to know how the new DLS 2.0 is different from the previous DLS. 
What were the major changes, and what was the previous version?

**Comment by moiz-conceptlz** [comment]
Hello @dotdesign 

The UAE Design System 2.0 is a complete overhaul. 

The most significant change is that the first design system version was simply a design - delivered in the form of a design file which still had gaps. The design system 2.0 has a lot more detailed guidelines, a design system with variables in Figma and an NPM package to use for code. 

The most significant change is the fact that the design system is now a package that can be installed for your front-end development, thereby creating very consistent code across all websites. Moreover, the package itself goes through constant improvement and updates that can directly reflect on websites using the package.

Every component has been carefully accessed, improved and coded to meet web standards.

I hope this is helpful.
